tools/hangar/__snapshots__/test_corpus/sdk_tests/bucket/rename.test.w_test_sim.md

```diff
@@ -2,7 +2,6 @@
 
 ## stdout.log
 ```log
-[ERROR] rename() | Copy (srcKey=no-such-obj.txt to dstKey=new-name.txt). Error: Source object does not exist (srcKey=no-such-obj.txt).
```

Comment:
Yes, this is an internal error of the resource which is caught by the user, so it's actually a happy path and shouldn't pollute the logs with red errors. If it weren't thrown, it would be shown as an error.